### PR TITLE
Update build badges to use Azure Pipeline Badges and links

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

Update build badges to use Azure Pipeline Badges and links
  * I also formatted tables

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Review by bergmeister — APPROVED** [review]
LGTM. At some point, the `.vsts-ci` would need renaming to azure pipelines as well (but this would also be a breaking change for the build definitions that would need updating then, which could complicate patch releases)

**Comment by TravisEz13** [comment]
@bergmeister If you want to rename the directory, please go ahead.  It won't affect our release process, but please mention me as I'll have to update the build definition is Azure Pipelines to point to the new location.

**Review by adityapatwardhan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -152,34 +152,37 @@ If you have any problems building, please consult the developer [FAQ][].
...
+|:-------------------------|:-----------------------------------------|:-------------------------------------|:-------------------------------------|:-------------------------------------|
+| [![av-image][]][av-site] | [![az-windows-image][]][az-windows-site] | [![az-linux-image][]][az-linux-site] | [![az-macos-image][]][az-macos-site] | [![az-spell-image][]][az-spell-site] |
 
 ### Build status of nightly builds
 
-| AppVeyor (Windows)       | Travis CI (Linux) | Travis CI (macOS) | Code Coverage Status | Code Factor Status |
-|--------------------------|-------------------|-------------------|----------------------|--------------------|
-| [![av-nightly-image][]][av-nightly-site] | [![linux-nightly-image][]][tv-site] | [![macOS-nightly-image][]][tv-site] | [![cc-image][]][cc-site] | [![cf-image][]][cf-site] |
+| AppVeyor (Windows)                       | Azure CI (Linux)                               | Azure CI (macOS)                               | Code Coverage Status     | Code Factor Status       |
```
Comment (daxian-dbw) [inline README.md]: minor: maybe just `Azure (Linux)` and `Azure (macOS)` without `CI`? `CI` was from `Travis CI`.

Comment (daxian-dbw) [inline README.md]: minor: maybe just `Azure (Linux)` and `Azure (macOS)` without `CI`? `CI` was from `Travis CI`.

**Review by daxian-dbw — APPROVED** [review]
Left one minor comment.

**Pull request merged**